applicatn.py

```diff
@@ -34,7 +34,7 @@ def register():
         name = request.form.get("name")
         password = request.form.get("password")
         userdata = Users(username=name, password=password, timestamp=str(datetime.now()))
-        print(f"added the user {name}{password}******************************************************")
+        # print(f"added the user {name}{password}******************************************************")
         try:
             db.session.add(userdata)
```

Comment:
how are you indicating registration is succesfull or not?

Reply:
Hello Mam, it is mentioned in the registered.html page that registration is
successful. Tq